### DISCUSSION TITLE
Subscribe button icon opposite

### DISCUSSION THREAD

**Discussion by camsteffen** [community/community#4877] [Mobile]
The subscribe button icon should indicate the current status rather than what the status would be after clicking.

**Comment by ArtemSmaznov** [comment]
Agree - I find myself confused by those as well

**Comment by eliperkins** [comment]
Thanks for this feedback @camsteffen and @ArtemSmaznov!

For my understanding, what would be more clear would be if the button at the bottom of the triage sheet in issues/pull requests said "Subscribed" when you're subscribed to an issue to indicate the current state, rather than "Unsubscribe" as an action to take?

**Reply by ArtemSmaznov** [reply]
![Screenshot_20210810-211518.png](https://user-images.githubusercontent.com/31800376/128928921-b56a0e35-5302-4205-9e5e-bc735de86425.png)
This looks to me that I am not subscribed to the ticket. I feel it would be cleared if this icon showed the current state. I expect the see it with the cross when I am actually not subscribed

**Reply by eliperkins** [reply]
Ah, got it. Thanks for this feedback, @ArtemSmaznov !

It looks like this is a duplicate of https://github.com/github/feedback/discussions/4925 as well then. Thanks for that clarification.
